function tryCatch(fn, onException) {
  try {
    const result = fn();
    return result;
  } catch (error) {
    onException?.(error);
  }
}
export {
  tryCatch
};
//# sourceMappingURL=tryCatch.mjs.map